lib/pure/algorithm.nim

```diff
@@ -357,15 +400,14 @@ func sort*[T](a: var openArray[T],
   ## * `sort proc<#sort,openArray[T]>`_
   ## * `sorted proc<#sorted,openArray[T],proc(T,T)>`_ sorted by ``cmp`` in the specified order
   ## * `sorted proc<#sorted,openArray[T]>`_
-  ## * `sortedByIt template<#sorted,openArray[T]>`_
+  ## * `sortedByIt template<#sortedByIt.t,untyped,untypedd,openArray[T]>`_
```

Comment:
Wrong link. It should be:

```
#sortedByIt.t,untyped,untyped
```

Reply:
Ohh :-/ a716c94e0efa61088c094e7ef52c3f84de8aebfe